Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "dummy"
+version = "0.0.0"
+
+[lib]
+path = "empty.rs"
+
+[dependencies]
+url = {git = "https://github.com/servo/rust-url.git", rev = "fbe5e50316105482dcd53d2dabb148c445a5f4cd"}
+libc = {git = "https://github.com/rust-lang/libc.git", rev = "8a85d662b90c14d458bc4ae9521a05564e20d7ae"}
+log = {git = "https://github.com/rust-lang-nursery/log.git", rev = "7a6028666b6e21e8cd39203e1f1114afc44333ef"}
```

Comment:
I’m fine with using normal versions instead of git + rev - I think that’s preferable ?